README.md

```diff
@@ -93,6 +93,10 @@ After a successful build, you should have a `glslc` executable somewhere under
 the `$BUILD_DIR/glslc/` directory, as well as a `libshaderc` library somewhere
 under the `$BUILD_DIR/libshaderc/` directory.
 
+The default behavior on MSVC is to link with the static CRT. If you would like
```

Comment:
That is correct. Specifically it refers to which library your executable/library/dll will statically link to, the one that is actually the static library or the one that exports the symbols for the shared CRT. I find the nomenclature of "static CRT" to be much more clear.